lib/gui/index.html

```diff
@@ -38,7 +38,7 @@
       <svg-icon path="../../../assets/etcher.svg"
         width="83px"
         height="13px"
-        os-open-external="https://etcher.io"></svg-icon>
+        os-open-external="https://etcher.io?ref=etcher"></svg-icon>
```

Comment:
Yeah that makes more sense, especially considering we might add more links in the future. I'll incorporate that.